static/include/components/apis/generated/arm-table.md

```diff
@@ -0,0 +1,14 @@
+<!-- prettier-ignore -->
+Method Name | Description
+----------- | -----------
+[`GetEndPosition`](/components/arm/#getendposition) | Get the current position of the arm as a [Pose](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.Pose).
```

Comment:
I don't this this should link to the python Pose class. Maybe we need to remove links from descriptions if that's the only way

Reply:
Not a merge blocker. you can file a ticket to fix this later

Reply:
Now stripping links from these MD table files in https://github.com/viamrobotics/docs/pull/2984!